docs/data/advanced/Study-Setup.md

```diff
@@ -19,11 +19,9 @@ Brace yourself, this will be a lot. We need to make this easier in the future...
   - If you have any files documenting standard operating procedure for different sites add them to a folder named `SOPs`
 
 ## 3. Set up XNAT
+  If you haven't used XNAT before, find out how to do attend the training course, set up an account, and about our naming conventions [here](data/advanced/XNAT.md).
   - Add any new accounts for the PI and any research assistants/collaborators they're known to have (this can be skipped till later) 
```

Comment:
since we're switching the using the KCNI's XNAT, this line can be removed. we're no longer handling this